test/edu/ucsb/nceas/metacat/doi/datacite/EML2DataCiteFactoryIT.java

```diff
@@ -30,7 +30,8 @@
 public class EML2DataCiteFactoryIT {
     
     public static final String section = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>";
-    public static final String section0 = "<resource xmlns=\"http://datacite.org/schema/kernel-4\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:schemaLocation=\"http://datacite.org/schema/kernel-4 https://schema.datacite.org/meta/kernel-4.3/metadata.xsd\"><identifier identifierType=\"DOI\">";
+    public static final String section0 = "<resource xmlns=\"http://datacite"
+        + ".org/schema/kernel-4\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:schemaLocation=\"http://datacite.org/schema/kernel-4 https://schema.datacite.org/meta/kernel-4.5/metadata.xsd\"><identifier identifierType=\"DOI\">";
```

Comment:
With this change, the line is still wider than 120 chars. I'd recommend we choose -- either:
* leave this as one long line, or 
* reformat this and all the other long lines in this section to be < 120 char.